product_filters/security/ir.model.access.csv

```diff
@@ -1,3 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_product_price_ranges_manager,product_price_ranges_manager,model_product_price_ranges,group_product_price_ranges_manager,1,1,1,1
 access_product_price_ranges_user,product_price_ranges_user,model_product_price_ranges,,1,0,0,0
+access_product_brand_public,public.product.brand,product_brand.model_product_brand,base.group_public,1,0,0,0
```

Comment:
Yes. II think so..

add ACL here and then if you create another app that need brands (directly or indirectly) you will need acl there and so on... It is a proper way add ACL where the model is created, not where it is used (just for very special cases) which is not the case now.